Editor/Scripts/ProjectSettings/EditorServerSettingsTab.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using NUnit.Framework;
```

Comment:
Is NUnit supposed to be here?

Reply:
Yes, at least according to the original file I based this on. There is an Assertion in the Setup function